tests/device_tests/bitcoin/test_signtx.py

```diff
@@ -885,7 +902,9 @@ def attack_processor(msg):
                 run_attack = True
             else:
                 msg.tx.outputs[0].address_n = []
-                msg.tx.outputs[0].address = "mwue7mokpBRAsJtHqEMcRPanYBmsSmYKvY"
+                msg.tx.outputs[
+                    0
+                ].address = "mwue7mokpBRAsJtHqEMcRPanYBmsSmYKvY"  # 44h/1h/4h/0/2
```

Comment:
please move the comment one line up so that this bad formatting does not happen

Reply:
fixed in 3d507a0